blockchain/block/genesis_block.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2021 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package block
+
+import (
+	"sync"
+	"time"
+
+	"github.com/iotexproject/go-pkgs/hash"
+
+	"github.com/iotexproject/iotex-core/config"
+	"github.com/iotexproject/iotex-core/pkg/version"
+)
+
+// constants
+const (
+	GenesisHashMainnet = "ab7d006c1f7a9345ad05eef1b4f062814a176c25c7558052e18896844ee71edb"
+	GenesisHashTestnet = "84b5d7158407b85c07ba1bcac0699dedc86d22bbcb29b6d77a5f696ac35047fd"
+)
```

Comment:
Where do we use them? Where should we put them?

Reply:
GenesisHashMainnet is used in test to make sure the calculated genesis hash (with mainnet genesis timestamp) is correct
can be moved, will fix